lib/workflows/icinga_downtime.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime, timezone
+
+from structs.icinga.downtime_details import DowntimeDetails
+from structs.icinga.icinga_account import IcingaAccount
+
+from icinga_api import downtime
+
+
+def schedule_downtime(
+    icinga_account: IcingaAccount,
+    object_type: str,
+    name: str,
+    start_time: str,
+    duration: int,
+    comment: str,
+    is_fixed: bool,
+):
+
+    datetime_object = datetime.strptime(start_time, "%d/%m/%y %H:%M:%S").replace(
+        tzinfo=timezone.utc
```

Comment:
Ah we need to be careful with timezones here, this assumes UTC but during the summer people will be thinking in BST. We can either change this to BST/UTC (some sort of automatic detection if it exists in the date library) or (though it puts a burden on the user) update the description to explicitly say UTC

Reply:
Changed to convert a UK time input into utc which icinga uses